Gemfile.lock

```diff
@@ -424,7 +415,7 @@ GEM
       unf_ext
     unf_ext (0.0.7.6)
     unicode-display_width (1.6.0)
-    uniform_notifier (1.12.1)
+    uniform_notifier (1.13.0)
```

Comment:
### [uniform_notifier](http://rubygems.org/gems/uniform_notifier)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.12.1 → 1.13.0

[_[change-log](https://github.com/flyerhzm/uniform_notifier/blob/master/CHANGELOG.md), [source-code](https://github.com/flyerhzm/uniform_notifier)_]





<details>
<summary>Commits</summary>

A change of **11** commits. See the full changes on [the compare page](https://github.com/flyerhzm/uniform_notifier/compare/v1.12.1...v1.13.0).

These are the first 10 commits:
- (72743f9) [Add rubygems metadata](https://github.com/flyerhzm/uniform_notifier/commit/72743f94e7167ab2e81cbb209cc77babe7c0e242)
- (0186f0c) [Merge pull request #47 from viraptor/patch-1](https://github.com/flyerhzm/uniform_notifier/commit/0186f0c23ad504365c3b145a016e233a2927e80d)
- (6949382) [Auto corrected by following Ruby Style/IfUnlessModifier](https://github.com/flyerhzm/uniform_notifier/commit/69493822c9be2981fcba1572f441a561292edf72)
- (ed76d41) [Allow configuration of Rollbar level.](https://github.com/flyerhzm/uniform_notifier/commit/ed76d4138805c7b1ed0044bae5d24c87055f2a26)
- (cf357b7) [Merge pull request #48 from cabello/rollbar-level](https://github.com/flyerhzm/uniform_notifier/commit/cf357b7038bcc2f5716c973e148e4995b0d666fb)
- (f0efc9d) [Removed rubyforge_project from gemspec. Because rubyforge wa…](https://github.com/flyerhzm/uniform_notifier/commit/f0efc9d6a6907d2b4a5dfc5351286ca2f8ca9cc6)
- (cc91ae5) [Merge pull request #49 from hsbt/remove-rubyforge](https://github.com/flyerhzm/uniform_notifier/commit/cc91ae5072a4faef108113d51e55b5ef859b93d3)
- (4075952) [update README](https://github.com/flyerhzm/uniform_notifier/commit/40759520116516e1f9b3cf433745ed48828d4890)
- (87ec8ed) [Add Honeybadger class dependecy injection](https://github.com/flyerhzm/uniform_notifier/commit/87ec8ed66742dc42d9fa61f1839049a03aa30756)
- (ff24c79) [Merge pull request #54 from decisely/add-honeybadger-class-d…](https://github.com/flyerhzm/uniform_notifier/commit/ff24c790453cc38051fa8f146cec542b50ba4574)


</details>